src/Sarif.UnitTests/Readers/IsSuppressedInSourceConverterTests.cs

```diff
@@ -87,6 +89,9 @@ public void BaselineState_None()
                         BaselineState = BaselineState.None
                     }
                 });
+
+                uut.CloseResults();
```

Comment:
>CloseResults [](start = 20, length = 12)

note this alternate pattern, currently supported by code, to explicitly close results and write a tool object, rather than allowing the log writer Dispose method to handle it.

Reply:
That's fine. Just curious why you changed the test to use that pattern. If the reason was to get coverage on this code path, then your PR comment should become an in-source comment explaining that.

---
In reply to: [255249783](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255249783) [](ancestors = 255249783)

Reply:
Add some comments. If we act on the design ideas mentioned here for this writer, all these calls will go away in future.

---
In reply to: [255309635](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255309635) [](ancestors = 255309635,255249783)